### PR TITLE
Add UI validation to multi attribute login claim set

### PR DESCRIPTION
**Pull request by sandushi**
### Related Issues
- Resolved Issue https://github.com/wso2/product-is/issues/13284

### CODE REVIEW

File: apps/console/src/features/server-configurations/constants/server-configurations-constants.ts [hunk 1]
```diff
@@ -164,6 +164,14 @@ export class ServerConfigurationsConstants {
 	 */
 	public static readonly PASSWORD_POLICY_CONNECTOR_ID: string = "cGFzc3dvcmRQb2xpY3k";
 
+	/**
```
Comment (brionmario) [inline apps/console/src/features/server-configurations/constants/server-configurations-constants.ts:167]:
```suggestion
    /**
```

File: apps/console/src/features/server-configurations/constants/server-configurations-constants.ts [hunk 2]
```diff
@@ -321,6 +329,10 @@ export class ServerConfigurationsConstants {
 	public static readonly OFFLINE_PASSWORD_RESET: string = "Recovery.AdminPasswordReset.Offline";
 	public static readonly ADMIN_FORCED_PASSWORD_RESET_EXPIRY_TIME: string = "Recovery.AdminPasswordReset.ExpiryTime";
 
+    //multi attribute login set
```
Comment (brionmario) [inline apps/console/src/features/server-configurations/constants/server-configurations-constants.ts:332]: The const name is self explanatory IMO
```suggestion
```

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/server-configurations/constants/server-configurations-constants.ts (hunk 1 above)
Comment (brionmario) [reply apps/console/src/features/server-configurations/constants/server-configurations-constants.ts]: Fixed with https://github.com/wso2/identity-apps/pull/2968/commits/7f3543d519386823e8b0b00aa50ec13a97c71cad

File: apps/console/src/features/server-configurations/constants/server-configurations-constants.ts (hunk 2 above)
Comment (sandushi) [reply apps/console/src/features/server-configurations/constants/server-configurations-constants.ts:332]: fixed with https://github.com/wso2/identity-apps/pull/2978/commits/8a276d8dd97d945b5e955fb2c47e010cb23f6dc5

Comment (lashinijay) [inline apps/console/src/features/server-configurations/constants/server-configurations-constants.ts:332]: Shall we format this comment properly? Start with an uppercase letter and ends with a full stop.

File: apps/console/src/features/server-configurations/constants/server-configurations-constants.ts [hunk 3]
```diff
@@ -164,6 +164,14 @@ export class ServerConfigurationsConstants {
 	 */
 	public static readonly PASSWORD_POLICY_CONNECTOR_ID: string = "cGFzc3dvcmRQb2xpY3k";
 
+    /**
+    * Multi Attribute Login Claim List pattern regex.
+    * @constant
+    * @type {RegExp}
+    * @default
+    */
+    public static readonly MULTI_ATTRIBUTE_CLAIM_LIST_REGEX_PATTERN: RegExp = new RegExp("^(?:[a-zA-Z0-9:./]+,)+[a-zA-Z0-9:./]+$");
```
Comment (lashinijay) [inline apps/console/src/features/server-configurations/constants/server-configurations-constants.ts:173]: Is the indentation correct here?

**Pull request merged**